docs/source/en/pipeline_tutorial.mdx

```diff
@@ -33,101 +33,202 @@ While each task has an associated [`pipeline`], it is simpler to use the general
 ```py
 >>> from transformers import pipeline
 
->>> generator = pipeline(task="text-generation")
+>>> generator = pipeline(task="automatic-speech-recognition")
 ```
 
 2. Pass your input text to the [`pipeline`]:
 
 ```py
->>> generator(
-...     "Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone"
-... )  # doctest: +SKIP
-[{'generated_text': 'Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone, Seven for the Iron-priests at the door to the east, and thirteen for the Lord Kings at the end of the mountain'}]
+>>> generator("https://huggingface.co/datasets/Narsil/asr_dummy/resolve/main/mlk.flac")
+{'text': 'I HAVE A DREAM BUT ONE DAY THIS NATION WILL RISE UP LIVE UP THE TRUE MEANING OF ITS TREES'}
 ```
 
-If you have more than one input, pass your input as a list:
+Not perfect for what you had in mind ? Check out the models on the hub [Most downloaded models for automatic speech recognition](https://huggingface.co/models?pipeline_tag=automatic-speech-recognition&sort=downloads). 
+Let's take [openai/whisper-large](https://huggingface.co/openai/whisper-large) to try it out.
```

Comment:
I like the emphasis on "why" we want to change. It's because the output of the audio is incorrect towards the end.

Also the models are downloaded and it is a sort of ranking which is important. I don't care about model X downloaded 3 times and that handles nothing and was just a test experiment for X. I want to widely used common place models.

I feel those two informations are important.

Reply:
For sure, that makes sense!

I updated my suggestion a bit to keep those two important points. Mainly, I like making it super obvious to readers that we are talking about the result here and why they should check the most downloaded models. Feel free to ignore if you prefer it as is though 🙂